person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=with%20the%20following%3A-,Instance%20vars%3A%20%40id,-%2C%20%40name%2C%20and), Please add `id` as instance variable here in the `initialize` method.
[Hint] It will be something like that:
```ruby
def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
 end
```

Reply:
Changes made as per instructions